src/System.Security.Cryptography.OpenSsl/tests/RSAOpenSslProvider.cs

```diff
@@ -15,6 +15,11 @@ public RSA Create(int keySize)
         {
             return new RSAOpenSsl(keySize);
         }
+
+        public bool Supports384PrivateKey
+        {
+            get { return true; }
+        }
```

Comment:
Nit: You might consider starting to use expression-bodied members (though there's something to be said for consistency with the rest of the file), e.g.
``C#
public bool Supports384PrivateKey => true;
```